x-pack/plugins/alerting/server/rules_client/methods/enable.ts

```diff
@@ -83,13 +83,6 @@ async function enableWithOCC(context: RulesClientContext, { id }: { id: string }
     const updateAttributes = updateMeta(context, {
       ...attributes,
       ...(!existingApiKey && (await createNewAPIKeySet(context, { attributes, username }))),
-      ...(attributes.monitoring && {
-        monitoring: updateMonitoring({
-          monitoring: attributes.monitoring,
-          timestamp: now.toISOString(),
-          duration: 0,
-        }),
-      }),
```

Comment:
@XavierM Let's chat about this. I think there are 3 possible options for when this timestamp has to be updated, and I'd like to align on this semantics.

1. Only update it once the rule ends its execution (after each execution). This semantics corresponds to AFTER this fix.
2. Update it after each execution + when the rule gets enabled. This semantics corresponds to BEFORE this fix.
3. Update it after each execution + when the rule actually starts.

Reply:
We added that because we had the requirement to not show the last value of the duration after disabling/enabling. Maybe instead of doing what you offer we can change updateMonitoring function to that. So we do not lie to our user, we just reset.
```ts
export const updateMonitoring = ({
  monitoring,
  timestamp,
  duration,
}: {
  monitoring: RuleMonitoring;
  timestamp: string;
  duration?: number;
}) => {
  const { run } = monitoring;
  const { last_run: lastRun, ...rest } = run;

  return {
    run: {
      last_run: {},
      ...rest,
    },
  };
};
```

Reply:
Discussed offline, will change it according to @XavierM's suggestion!